src/scala/com/github/johnynek/bazel_deps/maven/Tool.scala

```diff
@@ -82,15 +83,16 @@ object Tool {
         })
       }
 
-      Dep(r(groupId), r(artifactId), r(version), scope)
+      Dep(r(groupId), r(artifactId), r(version), scope, classifier)
     }
   }
 
   def parseDep(e: Node): Dep =
     Dep(singleText(e \ "groupId"),
       singleText(e \ "artifactId"),
       singleText(e \ "version"),
-      optionalText(e \ "scope"))
+      optionalText(e \ "scope"),
+      optionalText(e \ "classifier"))
```

Comment:
did you test this at all with a maven pom file?

Reply:
I have not yet, but I will try to do that today.

Reply:
Do you have a known-working `pom.xml` file? I'm having running into a bunch of probably-tiny issues preventing me from running the tool at all, even on `master`. There were a bunch of little things that I sorta worked around, but the thing I'm running into right now is that apparently a scala version is mandatory.  -- Tool.scala:254 (`val scalaBinaryVersion = root.findProp("scala.binary.version").get`) blows up without it.

I'm attempting to run it in a pretty small, self-contained project because our monorepo apparently uses a bunch of maven features that aren't supported (we typically don't list `<version>` in subprojects; we only list `<packaging>` about 60% of the time, letting it default to `jar`). The scala thing might be the last of the issues,  but it's lunchtime, so it might be a bit before I get to find out :)